test/extensions/network/dns_resolver/cares/dns_impl_test.cc

```diff
@@ -723,6 +723,8 @@ class DnsImplTest : public testing::TestWithParam<Address::IpVersion> {
 
     cares.set_filter_unroutable_families(filterUnroutableFamilies());
     cares.set_allocated_udp_max_queries(udpMaxQueries());
+    cares.set_allocated_query_timeout_seconds(queryTimeoutSeconds());
```

Comment:
Seems that these tests always cover the cases where these options are set (either both to 0, or both to 10).
I think it would be better to have tests that don't set these options at all, and tests that do (and validate that the values are as expected in both cases).